integration/build.gradle

```diff
@@ -15,7 +15,10 @@ dependencies {
 	implementation 'org.junit.jupiter:junit-jupiter-api'
 
 	// Google Dependencies
-	implementation 'com.google.auth:google-auth-library-oauth2-http:1.5.3'
+	implementation('com.google.auth:google-auth-library-oauth2-http:1.5.3') {
+		exclude group: 'com.google.oauth-client', module: 'google-oauth-client'
+	}
+	implementation 'com.google.oauth-client:google-oauth-client:1.34.1'
```

Comment:
I'm curious how you knew to exclude `com.google.oauth-client`. (A high level explanation would work!)

Reply:
The links to the CVEs in the ticket told me! https://broadworkbench.atlassian.net/browse/ID-145

Reply:
cool! I think I'd be able to do this next time!